protocol/v2/ssv/validator/non_committee_validator.go

```diff
@@ -250,6 +257,13 @@ func (ncv *CommitteeObserver) processMessage(
 			}
 		}
 	}
+
+	// Remove older slots container
+	thresholdSlot := currentSlot - 12 // won't get new messages after 12 slots
+	if _, exists := ncv.postConsensusContainer[thresholdSlot]; exists {
```

Comment:
The linter complains we don't need this check (delete won't delete what doesn't exist)